### DISCUSSION TITLE
Add "Gothenburg University", OceanGliders Best Practices to readme?

### DISCUSSION THREAD

**Discussion by soerenthomsen** [GliderToolsCommunity/GliderTools#139] [General]
Dear GliderToolsCommunity, 

in the readme file it is written at the moment that GliderTools: "We provide examples and demonstrate best practices as developed by the SOCCO Group." 

I suggest to add "Gothenburg University" and something that we aim to implement Best Practices developed by OceanGliders at the moment and refer to the new [OceanGlidersCommunity](https://github.com/OceanGlidersCommunity) repo ? 
in particular the planed refactoring on oxygen will be largely based on this [OceanGliders Ocean SOP](https://github.com/OceanGlidersCommunity/Oxygen_SOP).

What do you think? @lukegre @callumrollo @isgiddy @tjryankeogh @tomhull @jbusecke @callumrollo 
@bastienqueste and others?

**Comment by bastienqueste** [comment]
Gothenburg and Voice of the Ocean...?

**Comment by isgiddy** [comment]
As new contributors I agree to add VOTO. I'll re-word for now (until we
align with the SOPs) so that the README has space to list all contributing
organisations as this grows. Thanks @Bastien Queste ***@***.***>
and @Callum Rollo ***@***.***> !

**Comment by jbusecke** [comment]
No particular opinion from my end. I think that others here have a better overview, and trust your judgement.
